site/src/analytics-releases.ts

```diff
@@ -0,0 +1,80 @@
+import { css, customElement, html, LitElement, property } from "lit-element";
+import { AnalyticsDataCurrent } from "../../worker/src/data";
+import "./components/analytics-chart";
+
+@customElement("analytics-releases")
+export class AnalyticsReleases extends LitElement {
+  @property({ attribute: false }) public currentData?: AnalyticsDataCurrent;
+
+  @property({ type: Boolean }) public isMobile = false;
+
+  @property({ type: Boolean }) public isDarkMode = false;
+
+  render() {
+    if (this.currentData === undefined) {
+      return html``;
+    }
+
+    const releases = Object();
+
+    Object.keys(this.currentData.versions)
+      .filter((key) => key.split(".").length === 3)
```

Comment:
No need to do the filtering there, you are throwing away everything after "x.x" 3 lines down

Reply:
The idea behind that line was to filter out all dev versions, e.g., `2021.6.0.dev20210511`.

Reply:
Those are nightly builds and should be counted the same way as 2021.3.4

Reply:
👌🏻 Wasn't sure about that. I removed the filter line